accord-core/src/main/java/accord/coordinate/tracking/QuorumTracker.java

```diff
@@ -66,6 +66,15 @@ boolean hasFailed()
         {
             return failures > shard.maxFailures;
         }
+
+        @Override
+        public String toString()
```

Comment:
not due to logging but more to debugging... we will have 30 to 60 shards and finding which ones to look at in a debugger is annoying without logging